cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -46,7 +46,8 @@ type MasterConfiguration struct {
 	// SelfHosted enables an alpha deployment type where the apiserver, scheduler, and
 	// controller manager are managed by Kubernetes itself. This option is likely to
 	// become the default in the future.
-	SelfHosted bool
+	SelfHosted     bool
+	SkipTokenPrint bool
```

Comment:
I think this is clearly a CLI thing, and should not be in the API for kubeadm itself.
The move of the selfhosted flag isn't relevant here, that was moved for other reasons.

What you could/should do here instead is wrap the args in a struct to avoid passing things as args to functions though.